guides/Implementierungsleitfaden-ISiK-Dokumentenaustausch-401/Einfuehrung/UebergreifendeFestlegungen/AkteureUndInteraktionen/Akteure.page.md

```diff
@@ -0,0 +1,28 @@
+---
+topic: AkteureUndInteraktionen-AkteureUndInteraktionen-Akteure
+---
+----
+
+## Akteure
+### Dokumentenserver
+Das bestätigungsrelevante System nimmt die Rolle des Dokumentenservers ein. Ein Dokumentenserver nimmt Dokumente von Clients zur Speicherung/Archivierung/Verwaltung entgegen und erlaubt Clients die Suche nach und den Abruf von Dokumenten.
```

Comment:
suggestion: hier Referenz auf Seite zu "bestätigungsrelevanten Systemen" hinzu, da hier ab Stufe 4 der "Dokumenten-Server" und "-Client" definiert werden

Reply:
zu 1) -> in die releasenotes einfügen (für Stufe 4 ) an der Stelle 3.0.0-rc
 zu 2) Abgrenzung zu ISiK Stufe 2 ( ->  "Stufe 2" entfernen